frame/evm/precompile/contracts/utils/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use darwinia_evm_precompile_utils_macro::selector;
+use evm::ExitError;
+
+const ACTION_LEN: usize = 4;
```

Comment:
Consider using https://github.com/darwinia-network/darwinia-common/blob/master/frame/support/src/evm.rs#L33. The same as `ACTION_LEN` here.

Reply:
move this here? This is only used in precompile contract.

Reply:
Just reuse it. It was used in the `transfer` precompile before.

Reply:
transfer precompile is also precompile contract? so it's better to put it in precompile/contracts/utils?

Reply:
the `transfer` is different from the two precompiles here. Just leave it there untouch.

Reply:
ok